nixos/modules/services/desktops/pipewire/wireplumber.nix

```diff
@@ -33,6 +35,12 @@ in
     ];
 
     environment.systemPackages = [ cfg.package ];
+
+    environment.etc.wireplumber."main.lua.d"."80-nixos.lua".text = if !pwUsedForAudio then ''
+      # Pipewire is not used for audio, so prevent it from grabbing audio devices
+      alsa_monitor.enable = function() end
```

Comment:
Is this documented anywhere? I do not see enable in https://pipewire.pages.freedesktop.org/wireplumber/configuration/alsa.html?highlight=alsa_monitor

Reply:
As far as I can see, it's not really, but what it's doing is basically just stubbing out this function call: https://gitlab.freedesktop.org/pipewire/wireplumber/-/blob/master/src/config/main.lua.d/30-alsa-monitor.lua